docs/resources/aide_conf.md.erb

```diff
@@ -3,32 +3,44 @@ title: The aide_conf Resource
 platform: linux
 ---
 
-# aide_conf
+# aide\_conf
```

Comment:
@kagarmoe I did not add code to affect escaping of underscores one way or another.  That is not the goal of this PR; and we had improperly escaped underscores sprinkled throughout. 

 However, here, it looks like the renderer *added* an escape where one was needed. Do you have an example of a place where one was removed?